org.omg.sysml/src/org/omg/sysml/expressions/functions/OrFunction.java

```diff
@@ -33,8 +33,8 @@ public String getOperatorName() {
 	}
 	
 	@Override
-	protected EList<Element> binaryBooleanOp(boolean x, boolean y) {
-		return EvaluationUtil.booleanResult(x | y);
+	protected EList<Element> binaryBooleanOp(Boolean x, Boolean y) {
+		return EvaluationUtil.booleanResult(x == Boolean.TRUE | y == Boolean.TRUE);
```

Comment:
Because this is the implementation of the unconditional '|' operator, so I used the equivalent Java operator (same for '&'). Admittedly, though, it doesn't really matter here.